activemq-artemis/templates/master-configmap.yaml

```diff
@@ -40,11 +41,10 @@ data:
     -N core="urn:activemq:core" \
     -u "/activemq:configuration/core:core/core:ha-policy/core:replication/core:master/core:group-name" \
     -v "${GROUPNAME}" /var/lib/artemis/etc-override/broker-11.xml
-  
-    echo Setting the connector-ref to ${HOSTNAME}
-    xmlstarlet ed -L \
-      -N activemq="urn:activemq" \
-      -N core="urn:activemq:core" \
-      -u "/activemq:configuration/core:core/core:cluster-connections/core:cluster-connection[@name='{{ $releaseName }}']/core:connector-ref" \
-      -v "${HOSTNAME}" /var/lib/artemis/etc-override/broker-10.xml
 
+    echo Setting the connector-ref to ${HOSTNAME}
+          xmlstarlet ed -L \
+            -N activemq="urn:activemq" \
+            -N core="urn:activemq:core" \
+            -u "/activemq:configuration/core:core/core:cluster-connections/core:cluster-connection[@name='{{ $releaseName }}']/core:connector-ref" \
+            -v "${HOSTNAME}" /var/lib/artemis/etc-override/broker-10.xml
```

Comment:
I'd leave a new line at the end of the file just in case a gnu gettext based utility has to read the file.

Reply:
Agreed. Nice catch.

Reply:
Updated in the latest commit

Reply:
Thanks